src/frequenz/dispatch/_actor_dispatcher.py

```diff
@@ -181,7 +181,7 @@ async def _start_actor(self, dispatch: Dispatch) -> None:
             if self._updates_sender is not None:
                 sent_str = ", sent a dispatch update instead of creating a new actor"
                 await self._updates_sender.send(dispatch_update)
-            _logger.warning(
+            _logger.info(
```

Comment:
I didn't get this change, the commit message says " Don't warn for changing dispatch parameters: It's a normal action ", does this mean that `_start_actor` is called every time a dispatch is updated? If so, I would even make this log a debug or remove it completely, because it seems misleading, as it was never the intention to start the actor if it was just a dispatch update.

If there is a way to tell if this is called because a dispatch just started or was updated, then maybe we can log more meaningful messages.

Reply:
I mean, there is a way, but it's not available to the actor_dispatcher (without restructuring).
The idea is that the dispatch instance tells us the desired state, no matter the previous state, so from that perspectives it doesn't matter whether it was an update or a new dispatch, both cases should do both, start or update a running actor instance..